Sources/Build/BuildDelegate.swift

```diff
@@ -235,6 +236,14 @@ public final class BuildDelegate: BuildSystemDelegate, SwiftCompilerOutputParser
     }
 
     public func commandStatusChanged(_ command: SPMLLBuild.Command, kind: CommandStatusKind) {
+        guard command.shouldShowStatus else { return }
+        guard !swiftParsers.keys.contains(command.name) else { return }
+        guard !isVerbose else { return }
+
+        queue.sync {
+            taskTracker.commandStatusChanged(command, kind: kind)
+            updateProgress()
+        }
```

Comment:
Should this be async? This is a hot callback, so if this takes any time to run it will slow down builds.

Reply:
You're correct. I'll switch them to `async`.